README.md

```diff
@@ -66,13 +66,13 @@ In your web browser, open the link [http://localhost:3000/](http://localhost:300
 
 ### Running applications with Docker
```

Comment:
Let's keep docker instructions from main branch, which was reviewed and approved by everyone

Reply:
ok